concepts/conditionals/introduction.md

```diff
@@ -1,46 +1,3 @@
-# Introduction
+# About
 
-Go contains basic numeric types that can represent sets of either integer or
```

Comment:
I'm not a go expert, but I think this document is about numbers, not conditionals, so I remvoed it.